db/migrate/20211117033947_change_email_to_be_citext_in_user.rb

```diff
@@ -0,0 +1,6 @@
+class ChangeEmailToBeCitextInUser < ActiveRecord::Migration[6.1]
+  def change
+    enable_extension("citext")
```

Comment:
@suho I think we should change in the original migration instead of creating new migration. New migration should be created when old migration has already been deployed on the server.

Reply:
@ankitkalia1195 Got your point, will update this in the next commit

Reply:
@ankitkalia1195 Fixed in [`d9e14f5` (#30)](https://github.com/suho/google-search-ruby/pull/30/commits/d9e14f55090f7da7fa08fb18edcd5e012d8cadbd)